dcs/src/test/jdbc_test/src/test/java/org/trafodion/jdbc_test/TestNetworkTimeout.java

```diff
@@ -64,10 +72,19 @@ public void testGetNetworkTimeout() {
 		Connection conn = null;
 		try {
 			System.out.println("Connecting to database...");
-			conn = Utils.getUserConnection();
+			conn = Utils.getUserConnection();S
```

Comment:
The extra "S" at the end of the line appears to be causing the JDBC build failure